public/index.html

```diff
@@ -2,20 +2,20 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="apple-touch-icon" sizes="180x180" href="%PUBLIC_URL%/apple-touch-icon.png">
-    <link rel="icon" href="%PUBLIC_URL%/favicon.svg">
-    <link rel="manifest" href="%PUBLIC_URL%/site.webmanifest" crossorigin="use-credentials">
-    <link rel="mask-icon" href="%PUBLIC_URL%/safari-pinned-tab.svg" color="#000000">
+    <link rel="apple-touch-icon" sizes="180x180" href="%PUBLIC_URL%/apple-touch-icon.png" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.svg" />
+    <link rel="manifest" href="%PUBLIC_URL%/site.webmanifest" crossorigin="use-credentials" />
+    <link rel="mask-icon" href="%PUBLIC_URL%/safari-pinned-tab.svg" color="#000000" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <meta name="msapplication-TileColor" content="#ffffff">
-    <meta name="theme-color" content="#ffffff">
-    <title>JoinMarket</title>
+    <meta name="msapplication-TileColor" content="#ffffff" />
+    <meta name="theme-color" content="#ffffff" />
+    <title>Jam</title>
```

Comment:
I think @theborakompanioni also brought this up: Should we say something like "Jam, for JoinMarket" here?

Reply:
Updated. ✅